doc/build_WIN.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1. Introduction
 
+> Update: for quick installation with package management tools in MSYS2 environment, please see [build_WIN_MSYS2.md](build_WIN_MSYS2.md)
```

Comment:
Why not replace all build guide with the new one? The old guide is really hard to follow with so many manual steps.

Reply:
There are still some gaps when running in MSYS2 environment such as numa specifying, let's figure them out.